example/ConquerClub.Domain/Events/MapInitialized.cs

```diff
@@ -0,0 +1,23 @@
+﻿using Qowaiv.Identifiers;
+
+namespace ConquerClub.Domain.Events
+{
+    public class MapInitialized
```

Comment:
The only event that is currently not a record it seems (not sure if there is a specific reason for that?).

Reply:
Left over, I will change that.